Documentation/teaching/labs/interrupts.rst

```diff
@@ -800,7 +800,8 @@ section before proceeding.
 
 Follow the sections maked with **TODO 2** in the skeleton.
 
-First, define an empty interrupt handling routine.
+First, define an empty interrupt handling routine with name
```

Comment:
Maybe use "named" instead of "with name".

Reply:
Agreed.  Just updated so.